---
layout: single
permalink: /project/
author_profile: false
title: "Project"
---

# FunChlam - The project

Chlamydiae, such as *Chlamydia trachomatis* or *C. pneumoniae*, are well known bacterial pathogens of humans and animals. Only in the past 25 years closely related bacteria have been found in diverse environments where they thrive as symbionts in protists or as pathogens in a variety of animal hosts. The discovery and analysis of these environmental chlamydiae has provided new, exciting, and valuable insights into the basic biology and evolution of the whole phylum Chlamydiae. Yet, our knowledge about these microbes is mainly derived from in-depth analysis of only few representatives. Molecular surveys though indicate a tremendous hidden diversity of chlamydiae in nature, and there is a remarkable gap concerning genomic information and isolates for the majority of environmental chlamydiae lineages. This project aims to characterize natural chlamydial communities. 

It is based on the following assumptions: 
1. Chlamydiae are ubiquitous in nature, but their abundance and community composition differ between habitats and are affected by seasonal shifts and host dynamics. 
2. The impact of chlamydiae on their protist hosts alters grazing behavior and thus contributes to shaping natural microbial communities and ecosystem functioning.
3.  In-depth analysis of novel environmental representatives of deeply branching chlamydial lineages will provide a more comprehensive overview over variations of chlamydial biology and the evolution of the host-associated lifestyle. 

Natural chlamydial communities from diverse environments will be investigated by combining a novel targeted amplicon sequencing assay, single cell and metagenomics, and isotope probing experiments with protist isolation approaches and microcosm experiments. The project will reveal the first comprehensive insights into chlamydial community structure and dynamics in nature. For the first time, the effect of the presence of chlamydiae on microbial food webs, community composition, and general microbial growth rate and mineralization will be determined. The recovery of novel chlamydiae and state-of-the-art single cell genomics will expand our knowledge about chlamydial biology, genome flexibility, and evolution. 
